keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -897,10 +1120,6 @@ type JobSpec struct {
   ...
 ```
 
-Note that, we do not introduce the `NotIn` operator in
```

Comment:
why remove this?

Reply:
It is a leftover note for the API where we could match pod conditions by the "operator and values" syntax. Now we provide a list of patterns, so the note is misleading.